### ISSUE TITLE
Repo is missing license

### ISSUE THREAD

**Issue by virtualzx1** [sleaze/rarbg-db-dumps#3]
The repo is missing a license, which makes it technically a closed sourced, even though I believe it is intended to be open source.  I suggest adding an MIT license

**Comment by douglasg14b** [comment]
Yeah, technically nothing on this repo can be used as it is implicitly copyrighted without a license.
